internal/datastore/mysql/migrations/zz_migration.0001_initial_schema.go

```diff
@@ -0,0 +1,18 @@
+package migrations
+
+const createMysqlMigrationVersion = `CREATE TABLE mysql_migration_version (
```

Comment:
The pg driver uses migration tables that are compatible with alembic. I don't see a big reason to maintain that compatibility but figured I'd highlight the difference just in case.